man/en/sos-report.1

```diff
@@ -226,7 +226,8 @@ Specify the number of threads sosreport will use for concurrency. Defaults to 4.
 .TP
 .B \--plugin-timeout TIMEOUT
 Specify a timeout in seconds to allow each plugin to run for. A value of 0
-means no timeout will be set.
+means no timeout will be set. The default value of -1 is a magic value used
+to indicate that the timeout is unset.
```

Comment:
Let's make this align with the verbiage in the option description - i.e. "A value of -1 is used to indicate the default timeout of 300 seconds".